src/app/legacy/components/RadioSchedule/testHelpers/helper.jsx

```diff
@@ -24,7 +24,7 @@ const getSchedule = (service, withLongSummary) => {
     service === 'arabic' ? 'المدة الزمنية %duration%' : 'Duration %duration%';
 
   return stateTypes.map((state, index) => ({
-    id: index,
+    id: `p${index}`,
```

Comment:
Would it be any easier to combine state & index as the id?
```suggestion
    id: `${state}_${index}`,
```

Reply:
OR just revert this...?

Reply:
We do need to make sure index is a string instead of a number, otherwise it has a console error in storybook. Adding the state there would make it a string so would be a better option.

Reply:
Having the p there though is more like the real thing - it is the episode number that becomes the id

Reply:
OK let's leave it as `p`